src/ImageProcessor.Web/Caching/DiskCache.cs

```diff
@@ -457,6 +460,49 @@ private static string GetValidatedAbsolutePath(string originalPath, out string v
         }
 
         /// <summary>
+        /// Returns a value indicating whether the requested image has been updated.
+        /// </summary>
+        /// <param name="creationDate">The creation date.</param>
+        /// <returns>The <see cref="bool"/></returns>
+        private async Task<bool> IsUpdatedAsync(DateTime creationDate)
+        {
+            bool isUpdated = false;
+
+            try
+            {
+                if (new Uri(this.RequestPath).IsFile)
+                {
+                    FileInfo imageFileInfo = new FileInfo(this.RequestPath);
```

Comment:
Can you call refresh on the File? I recall using FileInfo so I could ensure the info wasn't cached by the OS. Guess I don't really need to do that.